examples/visual-tests/amp-story/amp-story-shopping.html

```diff
@@ -48,80 +48,105 @@
           Example of Inline config. Multiple product tags with icon defined.
         -->
         <amp-story-shopping-attachment>
+          <script type="application/json">
+            {
+              "items" : [
+                 {
+                  "productUrl": "https://www.google.com",
+                  "productId": "lamp",
+                  "productTitle": "Brass Lamp",
+                  "productBrand": "Lamp Co",
+                  "productPrice": 799.00,
```

Comment:
Prices tend to be stored in backends as separate integers for dollar and cent values due to float arithmetic issues. When rendered, these are joined into a string.

I would allow string values here, validating them against a regexp to be number-like.

```js
if (typeof value === 'number') {
  // ok
} else  if (typeof value !== 'string' || !/^[0-9.,]+$/.test(value)) {
  // error
}
```

Reply:
Sounds good.